src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -57,11 +57,7 @@
 import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
 import org.glassfish.hk2.api.MultiException;
```

Comment:
But resteasy does not throw hk2-exceptions, but jax-rs exceptions, so that is different right?